app/javascript/application.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (<h1>Hello World!</h1>);
+}
+
+ReactDOM.render(
+  <App/>,
+  document.getElementById('root'),
```

Comment:
- [ ] Please create `App` and `Greeting` components in the `javascript` directory, These components should be responsible to render the greeting on the screen and also setup redux to manage the state. 

![image](https://user-images.githubusercontent.com/83803414/196521142-4b0e73ef-95bf-4168-a127-2b48d49fc6d8.png)